tensorflow/lite/micro/kernels/micro_ops.h

```diff
@@ -127,11 +127,16 @@ TFLMRegistration Register_UNIDIRECTIONAL_SEQUENCE_LSTM();
 TFLMRegistration Register_UNPACK();
 TFLMRegistration Register_VAR_HANDLE();
 TFLMRegistration Register_WHILE();
+TFLMRegistration Register_ZEROS_LIKE();
+
 // TODO(b/160234179): Change custom OPs to also return by value.
 namespace tflm_signal {
+TFLMRegistration* Register_RFFT();
+TFLMRegistration* Register_RFFT_FLOAT();
+TFLMRegistration* Register_RFFT_INT16();
+TFLMRegistration* Register_RFFT_INT32();
```

Comment:
let's move these to rfft.h, similar to https://github.com/tensorflow/tflite-micro/blob/1355b89368ff1168f77af1bde9b4569ef2e5b353/tensorflow/lite/micro/kernels/fully_connected.h#L71-L91

Reply:
Done.